src/pybind/mgr/dashboard/frontend/src/app/core/navigation/about/about.component.scss

```
@@ -1,43 +1,58 @@
 @use './src/styles/vendor/variables' as vv;
 
-.about-container {
+::ng-deep .cds--modal-container {
+  position: relative;
   background-color: vv.$secondary;
   background-image: url('../../../../assets/ceph_background.gif');
   background-position: right bottom;
   background-repeat: no-repeat;
   color: vv.$white;
   text-shadow: 1px 1px vv.$secondary;
-}
 
-.product-versions {
-  margin-top: 30px;
+  &::before {
+    content: '';
+    position: absolute;
+    top: 0;
+    left: 0;
+    width: 100%;
+    height: 100%;
+    background-color: vv.$secondary;
+    opacity: 0.1;
+    z-index: 0;
+  }
+
+  > * {
+    position: relative;
+    z-index: 1;
+  }
 }
 
-.product-versions strong {
-  margin-right: 10px;
+.header-content {
+  padding-left: 5rem;
+  padding-top: 2rem;
 }
 
-.modal-header {
-  border-bottom: 0;
+.product-versions {
+  margin-top: 0.875rem;
 }
 
-.modal-header .close {
-  color: vv.$white;
-  font-size: 2em;
+.product-versions strong {
+  margin-right: 0.625rem;
 }
 
 .modal-body {
-  padding-left: 80px;
-  padding-right: 80px;
+  padding-left: 5rem;
+  padding-right: 5rem;
+  color: vv.$white;
 }
 
 .ceph-logo {
-  margin-bottom: 30px;
-  width: 25%;
+  margin-bottom: 1.875rem;
+  width: 20%;
 }
 
-.modal-footer {
+::ng-deep.cds--modal-footer {
   border-top: 0;
   display: block;
-  padding: 15px 80px 35px;
+  padding: 0.938rem 6rem 2.188rem;
```

we don't need it to be accurate to the point. Btw can we use the carbon spacings instead?
```suggestion
  padding: 1rem 6rem 2rem;
```

---

The suggested changes have been made